application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -3922,6 +3927,20 @@ enum VertexType {
   TRANSIT
 }
 
+"Categorization for via locations."
+enum ViaLocationType {
+  """
+  Stop location must be visited on-board a transit vehicle or the journey must alight or board at
+  the location.
```

Comment:
```suggestion
  The via stop location must be visited as part of a transit trip as at the boarding stop, the intermediate stop, or the alighting stop.
```
The text is awkward, it is not clear that the "transit vehicle" is the same thing as the "journey".